VMBackup/main/Utils/DiskUtil.py

```diff
@@ -293,21 +297,30 @@ def get_mount_points(self):
                                 if(fstypeEnd >=0):
                                     fs_type = line[fstypeStart+1:fstypeEnd]
                             # If there is a duplicate, keep only the first instance
-                            if(mount_point not in mount_points):
-                                self.logger.log("mount command mount :" + str(mount_point) + ": and fstype :"+ str(fs_type) + ":", True) 
+                            if ((mount_point not in mount_points) and ((mount_point,deviceName,fs_type) not in mount_points_info)):
```

Comment:
Suggestion : as mount_points, mount_points_info seems same. Checking twice is not needed. i.e. check can be changed as below 
if ((mount_point,deviceName,fs_type) not in mount_points_info):

Reply:
if((mount_point,deviceName,fs_type) not in mount_points_info)
Above check alone will not be sufficient for the case where multiple devices were mounted on the same mount-point. Each combination will be different and all will be added to the list, which we have to avoid. Infact, rather (mount_point not in mount_points) alone is sufficient. So, keeping only that.